### PR TITLE
[WIP] Rackspace Migration Manager: Create FAQ article

### LINKED ISSUE #1839: [MVP] Rackspace Migration Manager: Create FAQ article
Use information from doc audit to create FAQ article - Internally Documented

Provide link to the PR for this issue to the PM, Trent Gillaspie, for review/additions.

### PR DESCRIPTION
**Pull request by cyrichardson**
Fixes #1839 

FAWS tab on how-to landing page will not go live until release planned for 3/7.

Will ask PM, Trent Gillaspie, to review. (He says he has a github account, trentgillaspie, but he is not showing up in the list of reviewers or as an @ mention. I'll ask him to confirm his account status.)

### CODE REVIEW

File: content/FAWS/migration-faq.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+to invest in it. We are simply making it easier for customers who want to
+migrate to AWS infrastructure to do so, while keeping the Rackspace expertise
+and support that they value. We also provide expertise and support for customers who use the Microsoft Azure public cloud and our OpenStack public
+cloud, as well as various private clouds and dedicated servers. We have no
+bias toward any of these infrastructure platforms and form factors, but
+instead help customers choose the ones that are best for each of their
+workloads. Ultimately, it is our customers who decide how many workloads we
+support on each platform.
+
+#### I heard that your sales representatives no longer available to new buyers of your public cloud. Is that correct?
```
Comment (trentgillaspie) [inline content/FAWS/migration-faq.md]: Should be, "I heard that your sales representatives ARE no longer available to new buyers of your public cloud. Is that correct?"

File: content/FAWS/migration-faq.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+No, that is not correct. We continue to sell our OpenStack public cloud
+through sales representatives and through our website.
+
+### Beta migration from Rackspace to AWS
+
+#### How can I take part in the beta migration program or learn more about it?
+
+Like any beta program, this one is limited in the number of customers it can
+accommodate. Customers who would like take part in the beta program or learn
+more about it should engage with their account manager who will work the
```
Comment (trentgillaspie) [inline content/FAWS/migration-faq.md]: "...account manager who will work WITH the..."

**Comment by trentgillaspie** [comment]
Added comments. Outside of those, looks great, @cyrichardson!

**Comment by cyrichardson** [comment]
@trentgillaspie Thank you!

File: content/FAWS/migration-faq.md (hunk 1 above)
Comment (cyrichardson) [inline content/FAWS/migration-faq.md]: Fixed

File: content/FAWS/migration-faq.md (hunk 2 above)
Comment (cyrichardson) [inline content/FAWS/migration-faq.md]: Fixed.

**Comment by cyrichardson** [comment]
@kmsholcomb Please provide a content edit for these FAQs when you can. Thank you!
@trentgillaspie Please let me know if you have any additional FAQs you'd like to add. Thank you!

**Comment by kmsholcomb** [comment]
Hi @cyrichardson. Suggested edits are in the attached Word doc and PDF copy of it. Not much to edit, really! But please let me know if you have any questions. Thanks!

[FAQ for migration from Rackspace to AWS_editsKH.pdf](https://github.com/rackerlabs/rackspace-how-to/files/772438/FAQ.for.migration.from.Rackspace.to.AWS_editsKH.pdf)
[FAQ for migration from Rackspace to AWS_editsKH.docx](https://github.com/rackerlabs/rackspace-how-to/files/772439/FAQ.for.migration.from.Rackspace.to.AWS_editsKH.docx)

**Comment by cyrichardson** [comment]
Thank you, @kmsholcomb!

**Comment by cyrichardson** [comment]
Ready to merge

**Comment by stephamon** [comment]
Release in May

**Comment by stephamon** [comment]
Release now mid-to-late June

**Comment by cyrichardson** [comment]
PR #2086 replaces this PR.
Content moved from FAWS folder to rackspace-migration-manager folder

**Pull request closed without merging**